cypress/fixtures/UMLS_API.txt

```diff
@@ -0,0 +1 @@
+fb766dc8-c0f8-4bc4-bb7f-0c924a208da0
```

Comment:
Similar comment as before, I don't think we will be needing this

Reply:
no longer using file for this API key value